interp/src.go

```diff
@@ -307,3 +309,13 @@ func isPathRelative(s string) bool {
 	p := "." + string(filepath.Separator)
 	return strings.HasPrefix(s, p) || strings.HasPrefix(s, "."+p)
 }
+
+// findSandboxPath checks if given import path belongs to some sandbox.
+func findSandboxPath(sandboxes map[string]string, importPath string) (string, bool) {
+	for k, v := range sandboxes {
+		if strings.HasPrefix(importPath, k+"/") {
```

Comment:
Potentially it could be worth the clean the import path before prefix comparison (i.e. `path.Clean(importPath)`). This may help mitigate path traversal vulnerabilities should users try to escape the file system sandbox root directory.

For instance, given:

```go
 i := interp.New(interp.Options{ Sandboxes: map[string]string{ "ext": "extensions", }, })
```

The importPath `"k/../foo"` would pass the prefix check and result in the path `"extensions/../foo"` -> `"foo"` being returned.

I may be missing some details, just wanted to post this for consideration while the PR is active.

Cheers,
Robin